MapboxNavigation/Resources/Base.lproj/Localizable.strings

```diff
@@ -136,10 +136,12 @@
 /* Accessibility value of label indicating the absence of a rating */
 "NO_RATING" = "No rating set.";
 
-/* General category of route feedback where user position is incorrect.
-   Specific route feedback that user positioning is incorrect. */
+/* General category of route feedback where user position is incorrect. */
 "POSITIONING" = "Positioning";
 
+/* Specific route feedback that user positioning is incorrect. */
+"POSITIONING_USER" = "User Position";
```

Comment:
It’s a bit weird to address the user as “User”. “Your position” would be more intuitive (and also consistent in terms of capitalization).

Reply:
Good point.

Reply:
Remember to rerun extract-localizable.sh to pick up the change to the string’s base value in the source code.